linkis-orchestrator/linkis-orchestrator-core/src/main/scala/com/webank/wedatasphere/linkis/orchestrator/plans/unit/CodeLogicalUnit.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 WeBank
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.webank.wedatasphere.linkis.orchestrator.plans.unit
+
+
+
+
+import java.util
+
+import com.webank.wedatasphere.linkis.manager.label.entity.engine.CodeLanguageLabel
+
+import scala.collection.JavaConverters._
+
+
+/**
+  *
+  * 代码存储单位，并通过CodeLanguageLabel判断是为sql or py等
```

Comment:
Please translate comments into English. And additional Chinese comments can be added if the logic is hard to understand.